coralproject/talk
# [CORL-2456] fix critical vulnerabilities in production

## What does this PR do?
This PR removes critical vulnerabilities in our production build

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. run a fresh `npm install` after checking out this branch
2. run `npm audit --omit=dev` and verify that there are no vulnerabilites marked as 'critical'
